src/lib/spinel/radio_spinel.hpp

```diff
@@ -1061,26 +1222,35 @@ template <typename InterfaceType> class RadioSpinel
     otRadioFrame  mAckRadioFrame;
     otRadioFrame *mTransmitFrame; ///< Points to the frame to send
 
-    otExtAddress mExtendedAddress;
-    uint16_t     mShortAddress;
-    uint16_t     mPanId;
-    otRadioCaps  mRadioCaps;
-    uint8_t      mChannel;
-    int8_t       mRxSensitivity;
-    otError      mTxError;
-    char         mVersion[kVersionStringSize];
-    otExtAddress mIeeeEui64;
+    otExtAddress        mExtendedAddress;
+    uint16_t            mShortAddress;
+    uint16_t            mPanId;
+    uint8_t             mChannel;
+    int8_t              mRxSensitivity;
+    otError             mTxError;
+    static char         sVersion[kVersionStringSize];
+    static otExtAddress sIeeeEui64;
+    static otRadioCaps  sRadioCaps;
 
     State mState;
-    bool  mIsPromiscuous : 1;     ///< Promiscuous mode.
-    bool  mIsReady : 1;           ///< NCP ready.
-    bool  mSupportsLogStream : 1; ///< RCP supports `LOG_STREAM` property with OpenThread log meta-data format.
-    bool  mIsTimeSynced : 1;      ///< Host has calculated the time difference between host and RCP.
+    bool  mIsPromiscuous : 1; ///< Promiscuous mode.
+    bool  mIsTimeSynced : 1;  ///< Host has calculated the time difference between host and RCP.
+
+    static bool sIsReady;           ///< NCP ready.
+    static bool sSupportsLogStream; ///< RCP supports `LOG_STREAM` property with OpenThread log meta-data format.
```

Comment:
Curious, why do we need these members `static`? Considering it will be on host.

Reply:
Same case as above. If all interfaces operate under the same process only one instance needs to handle this.